tests/test_simple.py

```diff
@@ -25,6 +25,28 @@
 
 
 class LdapSimpleTestCase(unittest.TestCase):
+    @defer.inlineCallbacks
+    def setUp(self):
+        self.ldap_server = yield create_ldap_server()
```

Comment:
it seems like we don't actually use this (or `auth_provider`) in the tests? so I'm not quite sure what its value is?

Reply:
Whoops, it was meant to simplify the tests so that we didn't have as much boiler plate....